app/api/v1/endpoints/auth.py

```diff
@@ -11,7 +11,7 @@
 @router.post("/register", response_model=UserInDB)
 @limiter.limit("50/minute")
```

Comment:
use a environment variable for constant for limit in minutes, if the limiter is common for all handlers add it as a global for all handlers